app/assets/stylesheets/syntax.scss

```diff
@@ -13,214 +13,220 @@ div.inner-comment div.body div.highlight pre.highlight {
   }
 }
 
+$code-highlight-background-color: #29292e;
+$code-highlight-text-color: #89c4f4;
```

Comment:
This one used to be very light color, like `#f8f8f2`, close to white. Now it's a shade of light blue which makes whole code block looks a bit too colorful IMO.

Reply:
@ludwiczakpawel Good point. Fixed.